velox/expression/tests/ExprTest.cpp

```diff
@@ -2956,14 +2956,14 @@ TEST_P(ParameterizedExprTest, tryWithConstantFailure) {
 TEST_P(ParameterizedExprTest, castExceptionContext) {
   assertError(
       "cast(c0 as bigint)",
-      makeFlatVector<std::string>({"1a"}),
+      makeRowVector({makeFlatVector<std::string>({"1a"})}),
```

Comment:
Would it be possible to extract this refactoring into a separate PR to allow reviewers to focus on material changes?